cmp/grouping/GroupingChooserModel.ts

```diff
@@ -284,12 +299,14 @@ export class GroupingChooserModel extends HoistModel {
         return {displayName: genDisplayName(src.name), ...src};
     }
 
-    getValueLabel(value: string[]) {
-        return value.map(dimName => this.getDimDisplayName(dimName)).join(' › ');
+    getValueLabel(value: string[]): string {
+        return (
+            value.map(dimName => this.getDimDisplayName(dimName)).join(' › ') ?? value.toString()
```

Comment:
Output of `map` will never be null - `?? value.toString()` not getting us anything here